src/TimeoutMigrationTool.SqlP.FakeData/TimeoutMigrationTool.SqlP.FakeData.csproj

```diff
@@ -0,0 +1,85 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is the old csproj format, copy the CS proj from another project?

Reply:
Done